lib/content/getting-started.md

```diff
@@ -232,7 +222,7 @@ module.exports = {
 
 Be sure to restart your server in order to pick-up this new code.
 
-If you cURL our new route (`curl http://localhost:3000/riddle-random`) or visit it in your browser, you'll see one of Paldo's riddles. We're up and running!
+If you cURL our new route (`curl http://localhost:3000/riddle-random`) or visit it [in your browser](http://localhost:3000/riddle-random), you'll see one of Paldo's riddles. We're up and running!
```

Comment:
Above there were some instructions removed to set the PORT back to 3000.  Does that mean we should use 4000 throughout the rest of the tutorial?

Reply:
I can't for the life of me recall what I was thinking there. I'll add some note about the rest of the tutorial serving on 3000 after that env setting section

Reply:
H'ok, added back a note explaining this away!